### ISSUE TITLE
Proposal text escapes diff view

### ISSUE THREAD

**Issue by camus-code** [decred/politeiagui#1149]
When looking through old proposal versions, it seems some of the styled markdown escapes the container. We need to make sure all text wraps within the container. 

<img width="987" alt="Screen Shot 2019-05-01 at 12 07 49 PM" src="https://user-images.githubusercontent.com/42082827/57030422-3a97a400-6c0a-11e9-9146-69cc3e4e3194.png">

**Comment by fernandoabolafio** [comment]
There are two issues here. One is the overflowing of code blocks as you mentioned. Second is this block of text should not be rendered inside a block of code. This should be just a paragraph.
See how it looks like in the proposal body: 
![Screen Shot 2019-05-02 at 12 28 18](https://user-images.githubusercontent.com/14864439/57069785-e2c66f00-6cd5-11e9-9569-d7ac6b02f653.png)

@victorgcramos you are more familiar with this diffing code. Do you have any thoughts on this?

**Comment by fernandoabolafio** [comment]
Here is the reason:

    this is a code block
- hello
    this is a paragraph


A text starting after 4 'blank spaces' is generally rendered as a code block, unless it is under/within a list. In that case, the content should be treated as a paragraph. In the regular markdown rendering, we have :
```html
<div className="md">
 // content
</div>
```

For the diff component we have many instances of the markdown, one inside of another:
```html
<div class="md">
   <div class="md">{html node}</div>
   <div class="md">{another html node}</div>
</div>
```
That's why the markdown in the diff always the text after 4 'blank spaces' as a code block, because it doesn't know about the list surrounding it since they are different markdown instances.

**Comment by victorgcramos** [comment]
@fernandoabolafio Sure! Taking a look now!

**Comment by fernandoabolafio** [comment]
closed on #1218
